src/components/Menu/index.tsx

```diff
@@ -51,9 +51,7 @@ const Menu = ({ username, loading }: MenuProps) => {
       {!loading && (
         <>
           <S.MenuGroup>
-            <S.IconWrapper>
-              <SearchIcon aria-label="Search" />
-            </S.IconWrapper>
+            <Search />
```

Comment:
Since we added the search here, we'll need to mock inside tests to not break the unit tests.

Reply:
Great point! I'll add the mock and push changes.